mgmtfn/k8splugin/README.md

```diff
@@ -0,0 +1,311 @@
+# Contiv Networking for Kubernetes
+
+Contiv is integrated with Kubernetes via a CNI plugin. This integration enables you
```

Comment:
minor: 'you' may sound too direct. Suggested text:
"This integration allows using Contiv Networking and Policy for Pod's interconnectivity in a Kubernetes cluster"